plugins/Login/tests/UI/ResetPassword_spec.js

```diff
@@ -129,6 +137,24 @@ describe('ResetPassword', function () {
         });
     });
 
+    describe('invalid username entered shows confirmation', function () {
+        this.title = parentSuite.title; // to make sure the screenshot prefix is the same
+
+        before(async function () {
+          // make sure we are not logged in
+          await page.goto('?module=Login&action=logout');
+          await page.waitForNetworkIdle();
+          await page.clearCookies();
+        });
+
+        it('should indicate a confirmation email might be sent even if username invalid', async function () {
+          await goToForgotPasswordPage();
+          await requestPasswordResetWithInvalidLogin();
+
+          expect(await page.screenshot({ fullPage: true })).to.matchImage('password_reset');
```

Comment:
I've been burnt several times already by trying to compare UI test screenshots to a previous image used in another test, because the second one would regenerate the processed screenshot again and then you lose the one from the previous comparison, making it harder to debug if there's an issue with the previous run.
Generally I think it's better to duplicate the expected screenshot with a specific name and not reuse them. E.g.

```suggestion
          expect(await page.screenshot({ fullPage: true })).to.matchImage('password_reset_invalid_user');
```

Reply:
Didn't think of that, updated